docs/servicedns-with-externaldns.md

```diff
@@ -42,7 +42,8 @@ providers.
 Setting-up MCSDNS can be accomplished by referencing the following documentation:
 
 - The Federation-v2 [User Guide](userguide.md) to setup one or more Kubernetes clusters and the Federation
-control-plane. If running in GKE, the cluster hosting the ExternalDNS controller must have scope
+control-plane. Due to [Issue #370](https://github.com/kubernetes-sigs/federation-v2/issues/370) you can not use minikube
```

Comment:
I think we can still use minikube here as long as we can enable the minikube support service type `LoadBalancer`, how about make the limitation to `NodePort` but not `minikube`?

Reply:
But after I install minikube, I can still deploy a helm chart like keepalived here https://github.com/gyliu513/work/tree/master/k8s/charts/keepalived and enable the minikube support `LoadBalancer` type, am I missing anything?

Reply:
one could also use [metallb](https://metallb.universe.tf/) to provision LoadBalancers in minikube clusters.